pkg/server/router/router.go

```diff
@@ -377,6 +383,13 @@ func (router *router) registerCloudAPIs(ws *restful.WebService) {
 	ws.Route(ws.GET("/clouds/{cloud}/ping").To(router.pingCloud).
 		Doc("Ping the cloud to check its health").
 		Param(ws.PathParameter("cloud", "name of the cloud").DataType("string")))
+
+	// GET /api/v1/clouds/{cloud}/workers
+	ws.Route(ws.GET("/clouds/{cloud}/workers").To(router.listWorkers).
+		Doc("Get all cyclone workers in the cloud").
+		Param(ws.PathParameter(cloudPathParameterName, "name of the cloud").DataType("string")).
+		Param(ws.HeaderParameter(cloudTypeHeaderName, "type of cloud").DataType("string")).
+		Param(ws.HeaderParameter(namespaceHeaderName, "namespace of kubernetes cluster").DataType("string")))
```

Comment:
Should be query param, not header param.

Reply:
done